game_eggs/steamcmd_servers/7_days_to_die/egg-7-days-to-die.json

```diff
@@ -1,13 +1,18 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v1",
+        "update_url": null
     },
-    "exported_at": "2019-11-06T20:06:59-05:00",
+    "exported_at": "2021-11-26T13:46:12+01:00",
     "name": "7 Days To Die",
     "author": "kristoffer.norman@bahnhof.se",
     "description": "7 days to die server",
-    "image": "quay.io\/parkervcp\/pterodactyl-images:ubuntu_source",
+    "features": null,
+    "images": [
+        "ghcr.io\/parkervcp\/games:source"
```

Comment:
Same as Ark. This image does not have rcon CLI for telnet and as such, breaks this egg.